documentation/functionalDoc/functionalDoc.tex

```diff
@@ -92,10 +92,38 @@ \subsection{Generators}
 \scriptsize{regulates the same node} & \multirow{-2}{*}{\scriptsize{true}} & \multirow{-2}{*}{\scriptsize{-}} & \scriptsize{+ VRRemote model} \\
 \bottomrule
 \end{tabular}
-\caption{Generators modelling.}
+\caption{Generators modelling when $targetV \leq TfoVoltageLevel$}
 \label{tab:generators_modelling}
 \end{table}
 
+\begin{table}[h!]
+\center
+\begin{tabular}{ c | c | c | c}
+\toprule
+\scriptsize{\textbf{{Voltage regulation}}} & \scriptsize{\textbf{{InfiniteReactiveLimits}}} & \scriptsize{\textbf{{Rectangular diagram}}} & \scriptsize{\textbf{{Model}}}\\
+\midrule
+\rowcolor{white}
+ \scriptsize{None}  & \scriptsize{-} & \scriptsize{-} & \scriptsize{fixed PQ generator} \\
+\rowcolor{gray!10}
+ \scriptsize{At the connection bus} & \scriptsize{false} & \scriptsize{false} & \scriptsize{GeneratorPVTfoDiagramPQSignalN} \\
+\rowcolor{white}
+ \scriptsize{At the connection bus} & \scriptsize{-} & \scriptsize{true} & \scriptsize{GeneratorPVTfoSignalN} \\
```

Comment:
Also the cases where infinite reactive limits = true

Reply:
the '-' means that this is true whatever the value of InfiniteReactiveLimits

Reply:
Ok, but maybe we could do the same for the other table ? What do you think ?